tools/benchmark/src/MochaReporter.ts

```diff
@@ -42,8 +42,9 @@ function getName(name: string): string {
  */
 // eslint-disable-next-line @typescript-eslint/no-extraneous-class
 module.exports = class {
-    public constructor(runner: Runner) {
-        const benchmarkReporter = new BenchmarkReporter();
+    public constructor(runner: Runner, options?: ReporterOptions) {
+        reporters.Base.call(this, runner, options);
```

Comment:
This looks straight-forward except for invoking the cctor function for `reporters.Base`.

@cqcmdwym - I was wondering if it would work to have this class extend 'reporters.Base' and call `super(runner, options)`?

Reply:
yes, if call super(runner, options),it will inherit mocha report power, however, maybe here needn't.